SpringProject/src/main/java/com/gradproject2019/userConference/controller/UserConferenceController.java

```diff
@@ -26,9 +26,15 @@
         return ResponseEntity.ok(newInterest);
     }
 
-    @GetMapping(path = "/user-conferences/{id}")
-    public ResponseEntity<List<ConferenceResponseDto>> getConferenceByUserId(@RequestHeader("Authorization") UUID token, @PathVariable("id") Long userId) {
+    @GetMapping(path = "/user-conferences/{userId}")
+    public ResponseEntity<List<ConferenceResponseDto>> getConferenceByUserId(@RequestHeader("Authorization") UUID token, @PathVariable("userId") Long userId) {
         List<ConferenceResponseDto> conferences = userConferenceService.getConferenceByUserId(token, userId);
         return ResponseEntity.ok(conferences);
     }
+
+    @DeleteMapping(path = "/user-conferences/{userId}/{conferenceId}")
```

Comment:
Here you access a resource by its key so it would be fine to have it as a path parameter, but it's a composite key and there's no way you can put those parameters next to each other. Doing it this way is wrong because you're not accessing the resource first by userId and then by conferenceId. Let's see what other helpers suggest.

Reply:
I had a look online and this seemed to be the suggested method, I wasn't sure it looked right though so if we do it differently I can do that